.gitignore

```
@@ -333,4 +334,4 @@ node_modules/
 /ajax/libs/*/*/**/custom
 /ajax/libs/*/*/**/README
 /ajax/libs/*/*/**/README.md
-/ajax/libs/*/*/**/README.markdown
+/ajax/libs/*/*/**/README.markdown
```

```suggestion
/ajax/libs/*/*/**/README.markdown

```
Please preserve newline :)